src/app/public/modules/checkbox/checkbox.component.ts

```diff
@@ -175,6 +178,17 @@ export class SkyCheckboxComponent implements ControlValueAccessor, Validator {
   public validate(control: AbstractControl): ValidationErrors {
     if (!this.control) {
       this.control = control;
+
+      // Get required state from AbstractControl.
+      // https://github.com/angular/components/issues/2574#issuecomment-461185408
+      const vf = this.control.validator(new FormControl());
```

Comment:
This would only get checked once on-load, correct? Maybe we should pull it out of the `if` statement so that it supports dynamic removal/addition of the `Validator.required` in the form control?

Reply:
Fixed. This should now update validation any time the input value changes.